MetaMask/metamask-extension
# Fix: multiple reject on the signature request screen

## Explanation



Currently, we show "Reject 2 transactions" on signature request screen when there are multiple requests to reject. 
With this PR we will show "Reject 2 requests".

## More Information


Fixes #15389

## Screenshots/Screencaps



### Before



![image](https://user-images.githubusercontent.com/97883527/195615641-cf7ace32-6d56-493c-a1d3-526fca9f9232.png)


### After



https://user-images.githubusercontent.com/97883527/195616578-343b5321-2eea-4231-a551-f9a8c80a4329.mov


## Manual Testing Steps



Use the test dapp to pop up a sign message prompt and don't sign it
Pop up the prompt again
See that it says "Reject 2 requests"

## Pre-Merge Checklist


### + If there are functional changes:
